.github/workflows/build.yml

```diff
@@ -0,0 +1,64 @@
+name: Build Connect Extension Utils
+
+on:
+  push:
+    branches: '*'
+    tags:
+        - '*'
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.7", "3.8", "3.9", "3.10"]
```

Comment:
3.7 is not needed 100%, maybe we can drop more.

Reply:
Perfect will drop 3.7 and 3.8